src/core/i18n/translator.js

```diff
@@ -7,17 +9,49 @@ export default class Translator {
    * @param {number} count The count associated with the pluralization
    */
   static translate (translations, interpolationParams, count) {
-    const stringToInterpolate = this._selectPluralization(translations, count);
+    const stringToInterpolate = this._selectStringToInterpolate(translations, count);
     return this._interpolate(stringToInterpolate, interpolationParams);
   }
 
-  static _selectPluralization (translations, count) {
+  /**
+   * If translations is json, parse it and choose the correct plural form.
+   * Otherwise just return the non-interpolated translation string.
+   * @param {string} translations
+   * @param {number} count
+   * @returns {string}
+   */
+  static _selectStringToInterpolate (translations, count) {
     try {
       translations = JSON.parse(translations);
-      return count > 1 ? translations.plural : translations['1'];
     } catch (e) {
       return translations;
     }
+    return this._selectPluralForm(translations, count);
+  }
+
+  /**
+   * Returns the correct plural form given a parsed translations object and count.
+   * @param {Object} parsedTranslations
+   * @param {number} count
+   * @returns {string}
+   */
+  static _selectPluralForm (parsedTranslations, count) {
+    let locale = parsedTranslations.locale;
+    if (!hasLang(locale)) {
+      locale = 'en';
```

Comment:
nit: We could make a default locale property in the Translator class and refer to that

Reply:
oh hmm right now the entire class is static so I think we would need to add a constructor and make this an instance method?